app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockScreen.kt

```diff
@@ -132,6 +132,17 @@ fun VaultUnlockScreen(
         state = rememberTopAppBarState(),
         canScroll = { !accountMenuVisible },
     )
+    val onAccountMenuActionClick: () -> Unit = remember(viewModel) {
+        {
+            // Ignore clicks when unlocking for FIDO 2 requests to prevent
```

Comment:
Instead of ignoring clicks, should we just hide the whole button.

Reply:
Good question. My thinking was showing the icon would help the user ensure they've selected the correct account. The text on screen is pretty explicit though, so it's not necessarily needed. I've asked for input from the design team. We'll see what they say. 👍

Reply:
I guess this may be a more invasive surgery but could we just make this change on the account action button itself: 
```kotlin
@Composable
fun BitwardenAccountActionItem(
    initials: String,
    color: Color,
    enabled: Boolean, // = !state.isUnlockingForFidoWoof
    onClick: () -> Unit,
) {
    val iconPainter = rememberVectorPainter(id = R.drawable.ic_account_initials_container)
    val contentDescription = stringResource(id = R.string.account)

    IconButton(
        enabled = enabled,
        onClick = onClick,
        colors = bitwardenStandardIconButtonColors(),
        modifier = Modifier.testTag("CurrentActiveAccount"),
    ) {

Reply:
I do agree that we should just disable the button if it still needed to be present. But the way we have done it everywhere else is to not show it at all, it seems a little odd to do it differently here.

Reply:
We'll be hiding the account icon during "woofer" operations. 🐕